control_plane/src/compute.rs

```diff
@@ -306,6 +306,9 @@ impl PostgresNode {
         conf.append("shared_preload_libraries", "neon");
         conf.append_line("");
         conf.append("neon.pageserver_connstring", &pageserver_connstr);
+        if let AuthType::NeonJWT = auth_type {
+            conf.append("neon.safekeeper_token_env", "$ZENITH_AUTH_TOKEN");
```

Comment:
This env variable is already set when running the compute node, it's used inside `pageserver_connstring`